### PR TITLE
Reposition the camera scan icon in the wallet > send functionality

### LINKED ISSUE #5784: Reposition the camera scan icon is the wallet > send
https://github.com/btcpayserver/btcpayserver/discussions/5366

This is low hanging fruit that would make using the wallet much better on mobile

271641565-5ea6596e-0947-416f-b114-74d1c7a93b96

### PR DESCRIPTION
**Pull request by TChukwuleta**
Resolves #5784 

Reposition the camera scan icon in the wallet > send functionality and also changing the camera icon to a scan qr icon.

### CODE REVIEW

**Review by dstrukt — APPROVED** [review]
LGTM!

cc @dennisreimann for code coverage

File: BTCPayServer/Views/UIWallets/WalletSend.cshtml [hunk 1]
```diff
@@ -94,8 +94,11 @@
                     <span class="fa fa-plus"></span> Add another destination
                 </button>
             </div>
-            <input asp-for="Outputs[0].DestinationAddress" class="form-control font-monospace" autofocus autocomplete="off" />
-            <span asp-validation-for="Outputs[0].DestinationAddress" class="text-danger"></span>
+            <div class="input-group">
+                <input asp-for="Outputs[0].DestinationAddress" class="form-control font-monospace flex-grow-1" autofocus autocomplete="off" />
+                <button type="button" id="scanqrcode" class="btn btn-secondary only-for-js" data-bs-toggle="modal" data-bs-target="#scanModal" title="Scan BIP21/Address with camera"><i class="fa fa-qrcode"></i></button>
+            </div>
+
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIWallets/WalletSend.cshtml:101]:
```suggestion
            <span asp-validation-for="Outputs[0].DestinationAddress" class="text-danger"></span>
```

File: BTCPayServer/Views/UIWallets/WalletSend.cshtml [hunk 2]
```diff
@@ -94,8 +94,11 @@
                     <span class="fa fa-plus"></span> Add another destination
                 </button>
             </div>
-            <input asp-for="Outputs[0].DestinationAddress" class="form-control font-monospace" autofocus autocomplete="off" />
-            <span asp-validation-for="Outputs[0].DestinationAddress" class="text-danger"></span>
+            <div class="input-group">
+                <input asp-for="Outputs[0].DestinationAddress" class="form-control font-monospace flex-grow-1" autofocus autocomplete="off" />
+                <button type="button" id="scanqrcode" class="btn btn-secondary only-for-js" data-bs-toggle="modal" data-bs-target="#scanModal" title="Scan BIP21/Address with camera"><i class="fa fa-qrcode"></i></button>
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIWallets/WalletSend.cshtml:99]:
```suggestion
                <button type="button" id="scanqrcode" class="btn btn-secondary only-for-js" data-bs-toggle="modal" data-bs-target="#scanModal" title="Scan BIP21/Address with camera">
                    <vc:icon symbol="scan-qr"/>
                </button>
```

**Review by dennisreimann — CHANGES_REQUESTED** [review]
Just some minor adaptions: Let's unify this to our custom icon and keep the input validation message.

**Review by dennisreimann — APPROVED** [review]
Adjusted the icon size, so that the stroke line doesn't look blurry. tACK

**Comment by dennisreimann** [comment]
grafik

File: BTCPayServer/Views/UIWallets/WalletSend.cshtml (hunk 1 above)
Comment (bwirsche) [reply BTCPayServer/Views/UIWallets/WalletSend.cshtml]: >how do I pull my own bitcoin

**Pull request merged**